tool/src/lib.rs

```diff
@@ -76,6 +78,30 @@ pub fn gen(
 
     match target_language {
         "js" => js::gen_bindings(&env, &mut out_texts, Some(docs_url_gen)).unwrap(),
+        "java" => {
+            let mut attr_validator = hir::BasicAttributeValidator::new("true");
+            attr_validator.support.renaming = true;
+            attr_validator.support.disabling = true;
+            attr_validator.support.iterators = true;
+            attr_validator.support.iterables = true;
+            attr_validator.support.indexing = true;
+            attr_validator.support.constructors = true;
+            attr_validator.support.named_constructors = true;
+            attr_validator.support.memory_sharing = true;
+            attr_validator.support.accessors = true;
```

Comment:
Just enabled everything for generating some code to start off with. Will adjust later